### PR TITLE
loss bug fixed

### PR DESCRIPTION
**Pull request by JingyuanHu**
There is a bug about regression_losses in FocalLoss. torch.stack will report : "Expected object of backend CUDA but got backend CPU for sequence element 0 in sequence argument at .... "  and crashed. Here is the bug fixed.

### CODE REVIEW

**Comment by yhenon** [comment]
thanks for your contribution
